dashboard/app/models/school.rb

```diff
@@ -56,6 +56,14 @@ def high_needs?
     stats.frl_eligible_total.to_f / stats.students_total.to_f > 0.5
   end
 
+  # Public school ids from NCES are always 12 digits, possibly with
+  # leading zeros. In the DB, those leading zeros have been stripped out.
+  # Other school types are less than 12 characters and in the DB they
+  # have not had their leading zeros removed.
+  def self.normalize_school_id(raw_school_id)
+    raw_school_id.length == 12 ? raw_school_id.to_i.to_s : raw_school_id
+  end
```

Comment:
Ick. Is this just in our DB? Any idea why it's like that? Does anything depend on it in this format? Would it be better to normalize our DB ids in a migration instead of coercing all future imports to match it in this function?

Reply:
I believe this is fallout from the ids originally being stored as integers and later converted to strings when we discovered that some schools have alphanumeric ids. The school id is the primary key on the table so anything that references the table depends on it and would also need updating. At the least, that is school_info and census submissions. I'm not sure what else. It would also mean changing all of the existing seeding/loading of school data. I'd like to do all that, but I'd need to carve out a lot of time to make it happen.